datadog/resource_datadog_dashboard.go

```diff
@@ -7613,6 +7795,343 @@ func buildDatadogTraceServiceDefinition(terraformDefinition map[string]interface
 	return datadogDefinition
 }
 
+//
+// Split Graph Definition helpers
+//
+
+func getSplitGraphDefinitionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"source_widget_definition": {
+			Description: "The original widget we are splitting on.",
+			Type:        schema.TypeList,
+			Required:    true,
```

Comment:
```suggestion
			Required:    true,
			MinItems:    1,
			MaxItems:    1,
```
should force this list to only be 1 element